addons/product/views/product_views.xml

```diff
@@ -48,13 +48,7 @@
                             </div>
                         </h1>
                     </div>
-                    <style>
-                        div[name="options"] .o_field_boolean > div {
-                            margin-left: 10px;
-                            margin-right: 0px;
-                        }
-                    </style>
-                    <div name="options">
+                    <div name="options" class="o_product_view">
```

Comment:
Was this the only module doing this kind of tricks?

Reply:
Regarding the checkboxes yes